app/supportAssignment/services/supportService.js

```diff
@@ -24,9 +24,11 @@ class SupportService {
 			},
 			updateSectionGroup: function (sectionGroup) {
 				return ApiService.put("/api/courseView/sectionGroups/" + sectionGroup.id, sectionGroup);
+			},
+			getAuditLogs: function (workgroupId, year) {
+				return ApiService.get("/api/workgroups/" + workgroupId + "/years/" + year + "/modules/Support Staff Assignments" + "/auditLogs");
```

Comment:
Not totally critical, but maybe assign the string to a variable, then pass `encodeURI(str)` to get()? Doesn't matter elsewhere seeing as how we're just using common characters, but we're now introducing spaces to REST...